fold-and-r.csl

```diff
@@ -119,8 +119,8 @@
         </else-if>
         <else-if type="article-journal">
           <group delimiter=" ">
-            <text macro="title" prefix="'" suffix="'"/>
-            <text term="in" suffix=" "/>
+            <text macro="title" quotes="true"/>
+            <text term="in" prefix=", " suffix=" "/>
```

Comment:
So the group delimiter occurs between all members of the group? (Told you I didn't know how this all works!)

So, either delete the delimiter, or I could make the delimiter a comma and remove the comma in the prefixes, right? Actually I think I could then just remove the prefixes altogether.